roles/deploy/tasks/deploy.yml

```diff
@@ -73,6 +73,16 @@
     - skip_ansible_lint
   notify: restart unicorn
 
+# Install JS dependencies with yarn
+
+- name: run yarn install
+  yarn:
+    path: "{{ current_path }}"
```

Comment:
the build is broken with:
TASK [deploy : run yarn install] ***********************************************
fatal: [local_travis]: FAILED! => {"changed": false, "msg": "Package.json does not exist in provided path."}
Monday 25 May 2020  14:44:15 +0000 (0:00:00.261)       0:10:33.399 ************ 

I wonder if I need to run this in the build_path instead.
Trying it in a new commit.

Reply:
yes, it fixes the build and it works.